test/checks/color/color-contrast.js

```diff
@@ -324,7 +324,7 @@ describe('color-contrast', function() {
 		assert.equal(checkContext._data.messageKey, 'shortTextContent');
 	});
 
-	it('should return true for a single character text with insufficient contrast', function() {
+	it('should return true for a single character text with sufficient contrast', function() {
```

Comment:
I think the description of this test is correct but the code setup is wrong. It seems to me that it should show that a failing contrast of a single character still passes the check (due to being ignored by short text content). Otherwise the test itself would pass as the contrast is sufficient.

Reply:
Changed so the test setup has failing contrast for the single character, and asserted that undefined is returned due to short text content. Let me know if that isn't what you were thinking.